app/scripts/components/common/mapbox/layers/raster-timeseries.tsx

```diff
@@ -385,6 +385,7 @@ export function MapLayerRasterTimeseries(props: MapLayerRasterTimeseriesProps) {
   // FitBounds when needed
   //
   useEffect(() => {
+    if (!stacCollection.length) return;
```

Comment:
@nerik I think now you removed the optional chaining altogether 🙃

Reply:
Yes, but the reason is:
<img width="875" alt="Screenshot 2022-10-19 at 20 26 55" src="https://user-images.githubusercontent.com/1583415/196774262-ddd7a4b4-ad60-47f5-8bdf-05c010bf5799.png">
<img width="845" alt="Screenshot 2022-10-19 at 20 28 36" src="https://user-images.githubusercontent.com/1583415/196774557-f6b28c71-99c6-4a94-bb5e-b0cff5f9eb8e.png">


...another rule triggered by `@typescript-eslint/strict` (`stacCollection` is a state initialized with an empty array)

Reply:
ooooh I see. Never mind then :)